docs/cloud-native-security/cloud-native-security-index.asciidoc

```diff
@@ -1,7 +1,39 @@
 [[cloud-native-security-overview]]
-
 = Cloud native security
-Elastic’s cloud security capabilities help you to improve your cloud security posture by comparing your cloud configuration to best practices, and allow you to monitor and investigate your Linux deployments inside and outside of Kubernetes.
+Elastic Security for Cloud helps you improve your cloud security posture by comparing your cloud configuration to best practices, and scanning for vulnerabilities. It also helps you monitor and investigate your cloud workloads inside and outside Kubernetes.
+
+This page describes what each solution does and provides links to more information.
+
+[discrete]
+== Cloud Security Posture Management (CSPM)
+Discovers and evaluates the services in your cloud environment — like storage, compute, IAM, and more — against configuration security guidelines defined by the https://www.cisecurity.org/[Center for Internet Security] (CIS) to help you identify and remediate risks that could undermine the confidentiality, integrity, and availability of your cloud data.
+
+To learn more, refer to <<cspm, Read the CSPM docs>>.
+
+[discrete]
+== Kubernetes Security Posture Management (KSPM)
+Allows you to identify configuration risks in the various components that make up your Kubernetes cluster.
+It does this by evaluating your Kubernetes clusters against secure configuration guidelines defined by the Center for Internet Security (CIS) and generating findings with step-by-step instructions for remediating potential security risks.
+
+<<kspm, Read the KSPM docs>>.
+
+[discrete]
+== Cloud Native Vulnerability Management (CNVM)
+Scans your cloud workloads for known vulnerabilities. When it finds a vulnerability, it supports your risk assessment by quickly providing information such as the vulnerability's CVSS and severity, which software versions it affects, and whether a fix is available.
+
+<<vuln-management-overview, Read the CNVM docs>>.
+
+[discrete]
+== Container Workload Protection
+Provides cloud-native runtime protections for containerized environments by identifying and (optionally) blocking unexpected system behavior in Kubernetes containers. These capabilities are sometimes referred to as container drift detection and prevention. The solution also captures detailed process telemetry from monitored containers, allowing you to set up custom alerts and protection rules.
+
+<<d4c-overview, Read the Container workload protection docs>>.
+
+[discrete]
+== Cloud Workload Protection for VMs
+Helps you monitor and protect your Linux VMs. It captures workload telemetry data containing process, file, and network activity. You can use this data — along with Elastic's out-of-the-box detection rules and machine learning models — to automate threat identification for your cloud VMs. Alerts that are generated from these detections help reduce the time to identify and remediate threats.
```

Comment:
> It captures workload telemetry data containing process, file, and network activity.

I might suggest this change as this solution is built on Defend and thus has more capabilities than just streaming data:

> "It uses [Elastic Defend's endpoint capabilities](https://not-sure-what-link-is-best.com) to instantly detect and prevent malicious behavior and malware from impacting system security and captures workload telemetry data..."